nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/services/azure/storage/ADLSCredentialsControllerService.java

```diff
@@ -50,9 +50,8 @@
 
     public static final PropertyDescriptor ACCOUNT_NAME = new PropertyDescriptor.Builder()
             .fromPropertyDescriptor(AzureStorageUtils.ACCOUNT_NAME)
-            .description(AzureStorageUtils.ACCOUNT_NAME_BASE_DESCRIPTION)
+            .description(AzureStorageUtils.ACCOUNT_NAME_BASE_DESCRIPTION + AzureStorageUtils.ACCOUNT_NAME_SECURITY_DESCRIPTION)
             .required(true)
-            .expressionLanguageSupported(ExpressionLanguageScope.NONE)
```

Comment:
Are we sure this is enough to remove this line? I think the default is `NONE` when not specified.

Reply:
The builder is instantiated with `fromPropertyDescriptor(AzureStorageUtils.ACCOUNT_NAME)` so it inherits `AzureStorageUtils.ACCOUNT_NAME`'s properties, including the expression language scope.

Reply:
Oh good point, thanks @adenes !